scala/build.sc

```diff
@@ -46,19 +33,77 @@ trait Defaults extends ScalaJSModule with PublishModule with ScalafmtModule {
 
   def sonatypeUri = "https://s01.oss.sonatype.org/service/local"
 
-  def sonatypeSnapshotUri = "https://s01.oss.sonatype.org/content/repositories/snapshots"
+  def sonatypeSnapshotUri =
+    "https://s01.oss.sonatype.org/content/repositories/snapshots"
+}
+
+object acsets extends Defaults {
+  def desc = "A flexible category theoretic in-memory database"
+
+  def ivyDeps = Agg(
+    ivy"com.lihaoyi::upickle::3.1.3",
+    ivy"org.typelevel::cats-core::2.10.0",
+    ivy"org.typelevel::cats-kernel::2.10.0"
+  )
+
+  def artifactName = "acsets"
+
+  object test extends ScalaTests {
+    def ivyDeps = Agg(ivy"com.lihaoyi::utest::0.8.1")
+
+    def testFramework = "utest.runner.Framework"
+  }
+}
+
+object balloons extends Defaults {
```

Comment:
We probably don't want this, because we don't have balloons, or acsets as a separate package...